x/concentrated-liquidity/msg_server_test.go

```diff
@@ -29,29 +28,13 @@ func (suite *KeeperTestSuite) TestCreateConcentratedPool_Events() {
 			denom0:                   ETH,
 			denom1:                   USDC,
 			tickSpacing:              DefaultTickSpacing,
-			exponentAtPriceOne:       DefaultExponentAtPriceOne,
 			expectedPoolCreatedEvent: 1,
 			expectedMessageEvents:    4, // 1 for pool created, 1 for coin spent, 1 for coin received, 1 for after pool create hook
 		},
 		"error: missing tickSpacing": {
-			denom0:             ETH,
-			denom1:             USDC,
-			exponentAtPriceOne: DefaultExponentAtPriceOne,
-			expectedError:      fmt.Errorf("tick spacing must be positive"),
-		},
-		"error: precision value below minimum": {
-			denom0:             ETH,
-			denom1:             USDC,
-			tickSpacing:        DefaultTickSpacing,
-			exponentAtPriceOne: cltypes.ExponentAtPriceOneMin.Sub(sdk.OneInt()),
-			expectedError:      cltypes.ExponentAtPriceOneError{ProvidedExponentAtPriceOne: cltypes.ExponentAtPriceOneMin.Sub(sdk.OneInt()), PrecisionValueAtPriceOneMin: cltypes.ExponentAtPriceOneMin, PrecisionValueAtPriceOneMax: cltypes.ExponentAtPriceOneMax},
-		},
-		"error: precision value above maximum": {
-			denom0:             ETH,
-			denom1:             USDC,
-			tickSpacing:        DefaultTickSpacing,
-			exponentAtPriceOne: cltypes.ExponentAtPriceOneMax.Add(sdk.OneInt()),
-			expectedError:      cltypes.ExponentAtPriceOneError{ProvidedExponentAtPriceOne: cltypes.ExponentAtPriceOneMax.Add(sdk.OneInt()), PrecisionValueAtPriceOneMin: cltypes.ExponentAtPriceOneMin, PrecisionValueAtPriceOneMax: cltypes.ExponentAtPriceOneMax},
```

Comment:
We should be still validating that tick spacing is authorized. Please add error case where it is unauthorized

Reply:
Added here https://github.com/osmosis-labs/osmosis/pull/4957/commits/7dcc66cd8e4277c7aa2f037c9758b22f7e4ec89d